Gemfile

```diff
@@ -1,3 +1,3 @@
 source 'https://rubygems.org'
 
-gem 'csvlint', github: 'GSS-Cogs/csvlint.rb', :ref => 'f859165bee3e99212ab6a4afb3308212ce3f7cb0'
+gem 'csvlint', git: 'https://github.com/GSS-Cogs/csvlint.rb'
```

Comment:
I quite like the fact there's a reference here. It means that when I want to debug a problem in the CSVLint docker container, I can quickly see exactly which version of the code we're using so I can investigate further. If you remove that then it makes it more difficult to tell whether the docker container has been deployed with a given fix. 

I know hashes aren't a great identifier, and I guess we could start using git tags and that would be a bit more pleasant to work with, but I don't think it's a good idea to remove the reference here.

Reply:
I have created a new tag in csvlint gem repository and now this tag is referenced in Gemfile.

Reply:
Looks good.